### PR TITLE
feat(p0): CHIT Completion — Consciousness canonical signing, Evo sign+verify, A2UI signature gate

### PR DESCRIPTION
**Pull request by POWERFULMOVES**
## What

Closes the **P0 CHIT Completion** lane from `ARCHON_ENHANCEMENT_ROADMAP.md` §3 (claimed in #2079): the three Partial-tier CHIT services now sign/verify through the canonical single source of truth.

### Consciousness (8106)
- The service had its **own drifted inline signer** (hash-derived `kid`, extra `ts` field) — replaced with the canonical `pmoves.tools.chit_security` import. A standalone fallback remains for images that don't package `pmoves.tools` (pairs with #2070), aligned **byte-for-byte** and locked by a fallback-parity test that compares signatures against the canonical signer.
- Key sourcing was `CHIT_PROD_PASSPHRASE` only (canonical deployments silently unsigned!) → now `CHIT_SIGNING_KEY` > `CHIT_PASSPHRASE` > legacy `CHIT_PROD_PASSPHRASE`.
- `CHIT_REQUIRE_SIGNATURE` fail-closed at startup, CHR publish, and mapper publish.
- **`cgp_mapper.publish_to_hirag` now signs at the publish boundary** — previously every `/cgp/generate|publish|batch` packet went to Hi-RAG unsigned.

### Evo Controller (8113) — from zero
- Signs `geometry.swarm.meta.v1` payloads before publish (via Agent Zero events API; wrappers from `services.common.geometry_decoder`).
- Verifies inbound CGP signatures in `_filter_verified_cgps`: **tampered packets always dropped**; unsigned + non-dict dropped under fail-closed.
- `/config` exposes `chit_signing_enabled` / `chit_signature_required`.

### A2UI NATS Bridge (9224) — consumer edge
- `cgp_passes_signature_gate()` in `geometry_handler`: with a key set, tampered geometry never reaches WebSocket clients; unsigned/unverifiable/non-dict gated by `CHIT_REQUIRE_SIGNATURE`; rejections observable via `a2ui_geometry_events_rejected_total{reason}`.

### Docs
- `CHIT_INTEGRATION_STATUS.md` rows reconciled (Consciousness was listed at port 8096 with a pre-CHR gap list — stale since 2026-03-24).
- Register RELEASE entry appended (may need trivial EOF-merge vs #2079's CLAIM).

## Testing
- **45 new tests, all green** (18 consciousness, 12 evo, 9 bridge + parity/edge cases): round-trip, tamper, wrong-key, key-chain precedence, dev-mode unsigned, fail-closed raise/refuse/drop, publish-boundary signing, fallback byte-parity.
- Pre-existing failure set (crypto-dep + persona_gate) proven **identical on clean main** — zero regressions.
- `chit-compliance-reviewer` verdict: **APPROVE** (6/6 PASS). Its P2 non-dict bypass fixed in `ed78da0c4`; P2 evo rejection metric deferred (service has no prometheus dep or /metrics endpoint — follow-up).

## Env contract (unchanged defaults, opt-in hardening)
Dev behavior without a key is unchanged (unsigned + warning). Set `CHIT_REQUIRE_SIGNATURE=true` per service to go fail-closed. No new NATS subjects.

Refs: #2079 (lane claim), #2070 (pmoves.tools packaging pairs with the consciousness canonical import).

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: pmoves/services/evo-controller/app.py [hunk 1]
```diff
@@ -47,6 +47,31 @@ def get_service_url_sync(slug: str, *, default_port: int = 80) -> str:
 except ImportError:
     NATS_ANNOUNCE_AVAILABLE = False
 
+# CHIT signing/verification — canonical wrappers from services.common
+# (single source of truth: pmoves.tools.chit_security). If the image does not
+# package pmoves.tools (see the service-tools packaging gate), signing degrades
+# to dev mode unless CHIT_REQUIRE_SIGNATURE forces fail-closed.
+try:
+    from services.common.geometry_decoder import sign_cgp, verify_cgp
```
[the remark this answers is not in the document]
Comment (POWERFULMOVES) [reply pmoves/services/evo-controller/app.py:55]: Fixed in e20306bc7 — the evo image now packages the pmoves namespace (chit_common/chit_security + services/common under pmoves/), verified in-container: `from services.common.geometry_decoder import sign_cgp, verify_cgp` succeeds in the built image. Note the chain also needs `pmoves.services.common.geometry_models` (a second package path beyond what this comment flagged) — included.

File: pmoves/services/a2ui-nats-bridge/bridge.py [hunk 2]
```diff
@@ -47,11 +47,64 @@
...
 
+# CHIT signature gate (consumer edge) — canonical verification via
+# services.common (single source of truth: pmoves.tools.chit_security).
+# The bridge is the last hop before geometry reaches the frontend; when a
+# signing key is available, tampered packets are dropped here. Unsigned
+# packets pass through unless CHIT_REQUIRE_SIGNATURE flips fail-closed.
+# Images that do not package the wrappers degrade to the historical
+# trust-passthrough behavior (dev mode).
+try:
+    from services.common.geometry_decoder import verify_cgp
```
[the remark this answers is not in the document]
Comment (POWERFULMOVES) [reply pmoves/services/a2ui-nats-bridge/bridge.py:58]: Fixed in e20306bc7 — a2ui build context flipped to pmoves/ (mirrors #2070's pattern for consciousness/tokenism), Dockerfile packages services/common + the pmoves namespace, split overlays regenerated. Verified in-container: the verifier import succeeds, so the signature gate is live rather than passthrough.
